app/controllers/CyaCategorisationController.scala

```diff
@@ -96,8 +96,6 @@ class CyaCategorisationController @Inject() (
             ).flatten
           )
 
-          //TODO this definitely will not survive in the new world
```

Comment:
why has this been removed - has it been fixed?

Reply:
There is a separate refactoring ticket for this